pkg/gen/awaitComments.go

```diff
@@ -107,7 +114,7 @@ Or (for Jobs): The Pod succeeded ('.status.phase' set to "Succeeded").
 2. The value of '.status.updateRevision' matches '.status.currentRevision'.
 `
 	default:
-		panic("unhandled kind: timeoutValues")
+		panic("comments: unhandled kind")
```

Comment:
Are we sure we want to panic here? It seems like an error is a better approach.

Reply:
It's just the generator code that runs at build time, so panic is fine here.